backend/data/tasks.ts

```diff
@@ -1,134 +1,134 @@
-import { Task } from './../server/core/interfaces/task.interface';
+import { TaskDocument } from './../server/core/interfaces/task.interface';
```

Comment:
Looks like you don't need this file anymore. If yes, please delete it. If no, please delete commented part.